src/applications/FlowrPhone/phoneWindow.ts

```diff
@@ -34,6 +35,65 @@ export class PhoneWindow extends KeyboardMixin(BrowserWindow) {
   private readonly _ipcEvents: {[key: string]: (...args: any[]) => void}
   private logger = getLogger('Phone window')
 
+  constructor(parent: BrowserWindow, preload: string | undefined, index: string, props: PhoneAppProps, private store?: Store<Record<string, any>> | undefined) {
+    super(Object.assign({
+      frame: false,
+      transparent: true,
+      show: false,
+      parent,
+      backgroundColor: '#00000000',
+      webPreferences: {
+        plugins: true,
+        nodeIntegration: false,
+        contextIsolation: false,
+        experimentalFeatures: true,
+        preload,
+      },
+    }, buildPositionFromParents(parent.getContentBounds())))
+
+    const pageUrl = new URL(index)
+
+    /* eslint-disable @typescript-eslint/no-floating-promises */
+    this.loadURL(pageUrl.href)
+
+    this.mode = WindowModes.WIDGET
+    /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+      this._ipcEvents = {
+      'phone-maximize': () => this.mode = WindowModes.FULLSCREEN,
+      'phone-reduce': () => this.mode = WindowModes.WIDGET,
+      'phone-show': this.show.bind(this),
+      'phone-hide': this.hide.bind(this),
+      'register-props': this.updateRegisterProps.bind(this),
+      'phone-mute': this.mute.bind(this),
+      setDebugMode: (evt: any, debugMode: boolean) => {
+        if (debugMode) {
+          this.webContents.openDevTools()
+        } else {
+          this.webContents.closeDevTools()
+        }
+      },
+      'update-phone-store': this.updateStore.bind(this),
+        initProps: () => {
+          this.webContents.send( 'init-props',
```

Comment:
Since it's not possible (as expected), let's rename this to
```suggestion
          this.webContents.send( 'initPropsReply',
```